lib/Twig/ContextParameterInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * For implementing when an additional logic is needed before
+ * pass the template context parameter to the template.
+ *
+ * @author Vladimir Balin <krocos@mail.ru>
+ */
+interface ContextParameterInterface
```

Comment:
the class name should be ``Twig_ContextParameterInterface``. Your interface is currently not autoloadable and polutes the global namespace outside Twig, making conflicts likely